### ISSUE TITLE
Freezes with v@node1:/sdd/GSTREAMER/gst-inference/gst-shark$ gst-launch-1.0 filesrc location=$VIDEO_FILE ! decodebin ! videoconvert ! videoscale ! queue ! net.sink_model inceptionv4 name=net model-location=$MODEL_LOCATION backend=tensorflow backend::input-layer=$INPUT_LAYER  backend::output-layer=$OUTPUT_LAYER  Setting pipeline to PAUSED ... 2019-04-02 13:16:53.378880: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA Pipeline is PREROLLING ... Redistribute latency... Pipeline is PREROLLED ... Caught SIGSEGV Setting pipeline to PL

### ISSUE THREAD

**Issue by gsrujana** [RidgeRun/gst-inference#155]
gst-launch-1.0 filesrc location=$VIDEO_FILE ! decodebin ! videoconvert ! videoscale ! queue ! net.sink_model inceptionv4 name=net model-location=$MODEL_LOCATION backend=tensorflow backend::input-layer=$INPUT_LAYER  backend::output-layer=$OUTPUT_LAYER 
Setting pipeline to PAUSED ...
2019-04-02 13:16:53.378880: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
Pipeline is PREROLLING ...
Redistribute latency...
Pipeline is PREROLLED ...
Caught SIGSEGV
Setting pipeline to PLAYING ...
New clock: GstSystemClock
Redistribute latency...
Spinning.  Please run 'gdb gst-launch-1.0 3791' to continue debugging, Ctrl-C to quit, or Ctrl-\ to dump core.
^Chandling interrupt.
Interrupt: Stopping pipeline ...
Execution ended after 0:04:14.362928966
Setting pipeline to PAUSED ...
Setting pipeline to READY ...
on using gstreamer inference with gshark tracer.. 
Ubuntu 16.04

**Comment by michaelgruner** [comment]
@GFallasRR @migueltaylor can you take a look at this? It looks the particularity of this pipeline is that there are no `src` pads in the inference element.

**Comment by GFallasRR** [comment]
I tested the same pipeline and it ran at my side, please check commands below.

export VIDEO_FILE=/home/cat.mp4
export MODEL_LOCATION=/home/graph_inceptionv4_tensorflow.pb 
export INPUT_LAYER='input'
export OUTPUT_LAYER='InceptionV4/Logits/Predictions'
#The pipeline does not have any display element, adding GST_DEBUG to check the output 
GST_DEBUG=*inception*:6 gst-launch-1.0 filesrc location=$VIDEO_FILE ! decodebin ! videoconvert ! videoscale ! queue ! net.sink_model inceptionv4 name=net model-location=$MODEL_LOCATION backend=tensorflow backend::input-layer=$INPUT_LAYER backend::output-layer=$OUTPUT_LAYER

=============================DEBUG_OUTPUT====================================
gfallas@gfallas-G5-5587:/home$ GST_DEBUG=*inception*:6 gst-launch-1.0 filesrc location=$VIDEO_FILE ! decodebin ! videoconvert ! videoscale ! queue ! net.sink_model inceptionv4 name=net model-location=$MODEL_LOCATION backend=tensorflow backend::input-layer=$INPUT_LAYER backend::output-layer=$OUTPUT_LAYER
Setting pipeline to PAUSED ...
2019-04-12 15:05:36.272577: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
0:00:01.188643347  1816       0xd68a70 INFO             inceptionv4 gstinceptionv4.c:308:gst_inceptionv4_start:<net> Starting Inception v4
Pipeline is PREROLLING ...
Redistribute latency...
Pipeline is PREROLLED ...
Setting pipeline to PLAYING ...
New clock: GstSystemClock
Redistribute latency...
0:00:01.228488636  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:208:gst_inceptionv4_preprocess:<net> Preprocess
0:00:02.812904929  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:277:gst_inceptionv4_postprocess:<net> Postprocess
0:00:02.812928382  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:298:gst_inceptionv4_postprocess:<net> Highest probability is label 282 : (0.636030)
0:00:02.812971662  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:208:gst_inceptionv4_preprocess:<net> Preprocess
0:00:02.996602922  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:277:gst_inceptionv4_postprocess:<net> Postprocess
0:00:02.996624197  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:298:gst_inceptionv4_postprocess:<net> Highest probability is label 282 : (0.654386)
0:00:02.996658732  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:208:gst_inceptionv4_preprocess:<net> Preprocess
0:00:03.176918040  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:277:gst_inceptionv4_postprocess:<net> Postprocess
0:00:03.176939625  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:298:gst_inceptionv4_postprocess:<net> Highest probability is label 282 : (0.659639)
0:00:03.176974896  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:208:gst_inceptionv4_preprocess:<net> Preprocess
0:00:03.358122337  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:277:gst_inceptionv4_postprocess:<net> Postprocess
0:00:03.358160043  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:298:gst_inceptionv4_postprocess:<net> Highest probability is label 282 : (0.657630)
0:00:03.358195696  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:208:gst_inceptionv4_preprocess:<net> Preprocess
0:00:03.545497339  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:277:gst_inceptionv4_postprocess:<net> Postprocess
0:00:03.545518816  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:298:gst_inceptionv4_postprocess:<net> Highest probability is label 282 : (0.658413)
0:00:03.545551458  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:208:gst_inceptionv4_preprocess:<net> Preprocess
0:00:03.727754331  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:277:gst_inceptionv4_postprocess:<net> Postprocess
0:00:03.727775046  1816       0x9d4000 LOG              inceptionv4 gstinceptionv4.c:298:gst_inceptionv4_postprocess:<net> Highest probability is label 282 : (0.654718)
--------------------------------------------------------------------------------------------------------------------------------------
@gsrujana can you share a log from this issue?
More information about your system will help to solve this problem, as example:
* GStreamer version
* TensorFlow version
* Branch tested
* Log files
* Hardware specifications
